explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -136,7 +136,11 @@ export function UpgradeableProgramSection({
             ) : (
               <>
                 {verifiableBuilds.map((b) => (
-                  <VerifiedBadge verifiableBuild={b} />
+                  <VerifiedBadge
+                    key={account.pubkey.toString()}
```

Comment:
This key should be unique for each element in the `verifiableBuilds` iteration

Reply:
I swapped to simply using the index in the array